docs/guide/topics/reactions.rst

```diff
@@ -0,0 +1,133 @@
+.. currentmodule:: discord
+
+.. _guide_topic_reactions
+
+Reactions
+==========
+
+Reactions are a feature within Discord where you can add emojis next to messages.
+They have many uses, but a common one for bots is for polls or giving feedback to a command without sending a new message.
+Anyone can react to a message.
+
+
+
+Adding a reaction
+~~~~~~~~~~~~~~~~~~
+
+Adding reactions is quite simple.
+
+To add a custom emoji reaction you can do the following:
+
+.. code-block:: python3
+
+    await message.add_reaction("<:amogus:819046807370989599>")
+
+Which would add a reaction to the message like so:
+
+.. image:: /images/guide/topics/reactions/add_custom_emoji.png
+    :scale: 50%
+
+
+.. note::
+
+    You must have the permission :attr:`~Permissions.read_message_history` to add reactions.
+    Additionally, if no one else has reacted to the message using this emoji, you will also need the
+    :attr:`~Permissions.add_reactions` permission.
+
+:meth:`~Message.add_reaction` can also take many types, not just :class:`str`. This includes :class:`~Emoji` or :class:`~PartialEmoji`.
+For example, the following shows an alternative way to add a custom emoji reaction:
+
+.. code-block:: python3
+
+    emoji = client.get_emoji(819046807370989599)
+    await message.add_reaction(emoji)
+
+You may notice that if you try to add a standard Discord emoji like ``:smile:``, you'll receive an error.
+This is because to add standard Discord emojis to a message, you must pass their unicode values.
+Because of this, to add 😄 / ``:smile:`` you would do either of the following:
```

Comment:
```suggestion
When adding standard (i.e. not custom) emojis as a reaction, you cannot pass emoji shortcodes,
such as the string ``":smile:"`` in order to react with 😄. The Discord client recognizes these
shortcodes as a typing convenience, but the API is not aware of them. So, in your bot, you must
pass the actual Unicode codepoint as is (e.g., ``"😄"``). This can be done like so:
```

This seems a bit pedantic, but the nomenclature can be ambiguous. "Emoji" can mean actual Unicode codepoints, Discord's custom emoji, or non-standard emoji shortcodes. I tried to reword stuff to make it less confusing here.

Reply:
`:name:` emojis are supported in other contexts as well- such as sending messages.

Reply:
Unless I'm missing something, shortcodes in messages don't seem to be postprocessed by the API in any way, so the remark "[...] the API is not aware of them" appears to hold:

![image](https://user-images.githubusercontent.com/4206232/158043226-5be8b8a8-ee4c-413a-927d-4b701e4605a6.png)

The client parses them though, of course.

Reply:
i do not see a reason to introduce client semantics to the guide.

Reply:
I feel like reducing the amount of potential confusion is desirable here, even at the expense of the result being a bit too wordy or technical. I don't think it's particularly difficult or overtly technical to explain that the API does not recognize shortcodes once it's spelled out, and that emojis are semantically distinct from shortcodes in this way.

I do see that this can be over the top here, though. Oh well pensive

Reply:
i feel like maybe a "the client and api see emojis in different ways, so you should be doing "X" to make sure you appease the api (in other wording of course).

Reply:
Oh, that could help, yeah. That's basically what you had in the first place, though, I think. So it's fine without my suggestion, seems.